impeller/toolkit/egl/image.h

```diff
@@ -0,0 +1,74 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#pragma once
+
+#include "flutter/fml/unique_object.h"
+#include "flutter/impeller/toolkit/egl/egl.h"
+
+namespace impeller {
+
+// Simple holder of an EGLImage and the owning EGLDisplay.
+struct EGLImageWithDisplay {
+  EGLImage image;
```

Comment:
Please default these to EGL_NO_IMAGE and EGL_NO_DISPLAY

Reply:
Good catch.

Done.